src/main/java/seedu/taskell/storage/JsonConfigStorage.java

```diff
@@ -0,0 +1,76 @@
+package seedu.taskell.storage;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+import seedu.taskell.commons.core.Config;
+import seedu.taskell.commons.core.LogsCenter;
+import seedu.taskell.commons.exceptions.DataConversionException;
+import seedu.taskell.commons.util.FileUtil;
+
+/**
+ * A class to access Config stored in the hard disk as a json file
+ */
+
+public class JsonConfigStorage implements ConfigStorage {
+
+    private static final Logger logger = LogsCenter.getLogger(JsonConfigStorage.class);
+    
+    private String filePath;
+    
+    public JsonConfigStorage(String filePath){
```

Comment:
Space is needed
![image](https://cloud.githubusercontent.com/assets/13817632/19619869/caff27ea-98a2-11e6-86db-64d98659bf66.png)